sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_conversation_app_async.py

```diff
@@ -30,47 +32,65 @@ async def sample_analyze_conversation_app_async():
     from azure.core.credentials import AzureKeyCredential
 
     from azure.ai.language.conversations.aio import ConversationAnalysisClient
-    from azure.ai.language.conversations.models import ConversationAnalysisOptions
+    from azure.ai.language.conversations.models import DateTimeResolution
 
     # get secrets
-    conv_endpoint = os.environ["AZURE_CONVERSATIONS_ENDPOINT"]
-    conv_key = os.environ["AZURE_CONVERSATIONS_KEY"]
-    conv_project = os.environ["AZURE_CONVERSATIONS_PROJECT"]
-
-    # prepare data
-    query = "One california maki please."
-    input = ConversationAnalysisOptions(
-        query=query
-    )
-
-    # analyze query
-    client = ConversationAnalysisClient(conv_endpoint, AzureKeyCredential(conv_key))
+    clu_endpoint = os.environ["AZURE_CLU_ENDPOINT"]
+    clu_key = os.environ["AZURE_CLU_KEY"]
+    project_name = os.environ["AZURE_CLU_CONVERSATIONS_PROJECT_NAME"]
+    deployment_name = os.environ["AZURE_CLU_CONVERSATIONS_DEPLOYMENT_NAME"]
+
+    # analyze quey
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     async with client:
-        result = await client.analyze_conversations(
-            input,
-            project_name=conv_project,
-            deployment_name='production'
+        query = "Send an email to Carol about the tomorrow's demo"
+        result = await client.analyze_conversation(
+            body={
+                "kind": "CustomConversation",
+                "analysisInput": {
+                    "conversationItem": {
+                        "participantId": "1",
```

Comment:
What's a participant ID?

Reply:
user or bot id in a dialogue
o means bot and 1 means user
in the April release which contains summarization, we can pass entire conversation as input with turns 0 and 1

Reply:
In that case, could we add some simply constants to _patch.py:
```
BOT_PARTICIPANT = "0"
USER_PARTICIPANT = "1"
```
Then update the samples to use these?

Reply:
We can open an issue to resolve this later